src/Private/Extras/String.elm

```diff
@@ -0,0 +1,9 @@
+module Private.Extras.String where
+
+maybeStringToString : Maybe String -> String
+maybeStringToString s =
+  case s of
+    Just s ->
+      s
+    Nothing ->
+      ""
```

Comment:
This is currently only used in one place. Either inline it or leave it factored out if it's used for later plots.